plugins/orchestrator/src/components/WorkflowInstancePage.tsx

```diff
@@ -26,15 +28,21 @@ export const WorkflowInstancePage = ({
     workflowInstanceRouteRef,
   );
 
-  const { loading, error, value, retry } = useAsyncRetry(async () => {
-    if (!instanceId && !queryInstanceId) {
-      return undefined;
-    }
-    return await orchestratorApi.getInstance(instanceId || queryInstanceId);
-  }, [orchestratorApi, queryInstanceId]);
+  const { loading, error, value, restart } = usePolling<
+    ProcessInstance | undefined
+  >(
+    async () => {
+      if (!instanceId && !queryInstanceId) {
+        return undefined;
+      }
+      return await orchestratorApi.getInstance(instanceId || queryInstanceId);
+    },
+    5000,
```

Comment:
Please extract this value to src/constants.ts

Reply:
done